srml/slashing/src/tests.rs

```diff
@@ -0,0 +1,161 @@
+#![cfg(test)]
+
+use super::*;
+use runtime_io::with_externalities;
+use srml_support::{decl_module, impl_outer_origin, dispatch::Result};
+use system::{self, ensure_signed};
+use substrate_primitives::{Blake2Hasher, H256};
+use primitives::{
+	BuildStorage,
+	traits::{BlakeTwo256, IdentityLookup},
+	testing::{Digest, DigestItem, Header}
+};
+
+impl_outer_origin! {
+	pub enum Origin for Test {}
+}
+
+type Balance = balances::Module<Test>;
+
+#[derive(Clone, Eq, PartialEq)]
+pub struct Test;
+
+impl system::Trait for Test {
+	type Origin = Origin;
+	type Index = u32;
+	type BlockNumber = u64;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type Digest = Digest;
+	type AccountId = u64;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = ();
+	type Log = DigestItem;
+}
+
+impl balances::Trait for Test {
+	type Balance = u64;
+	type OnFreeBalanceZero = ();
+	type OnNewAccount = ();
+	type Event = ();
+	type TransactionPayment = ();
+	type TransferPayment = ();
+	type DustRemoval = ();
+}
+
+impl OnSlashing for Test {
+	type Currency = Balance;
+	type Misconduct = Unresponsive;
+	type Slashing = Self;
+
+	fn slash(
```

Comment:
isn't this:
```
fn slash(who: AccountId, misconduct: Misconduct);
```
?

Reply:
yeah, this was `the old` impl but I have changed it and not updated the tests accordingly